Source/WebKit/UIProcess/RemoteLayerTree/RemoteLayerTreeDrawingAreaProxy.mm

```diff
@@ -339,25 +355,29 @@
     }
 }
 
-void RemoteLayerTreeDrawingAreaProxy::didRefreshDisplay()
+void RemoteLayerTreeDrawingAreaProxy::didRefreshDisplay(IPC::Connection* connection)
```

Comment:
Again, hard to know what the connection represents. And why doesn't this just fan out to all subframe processes?